server/admin_management/tests/services/test_user_validator.py

```diff
@@ -0,0 +1,177 @@
+import logging
+import pytest
+from fastapi import HTTPException
+from mock import patch
+
+from api.app.services.validator.user_validator import UserValidator, IdimProxyService
```

Comment:
Just curious to me, you can directly import "IdimProxyService" from `api.app.services.validator.user_validator`??, and use it for mock, wow!!

Reply:
oh yeah, I was surprised as well! but I changed the import to from the integration, feel that makes more sense